faststream/rabbit/broker/broker.py

```diff
@@ -346,23 +357,34 @@ async def _connect(  # type: ignore[override]
         *,
         timeout: "TimeoutType",
         ssl_context: Optional["SSLContext"],
+        max_connection_pool_size: int = 1,
+        max_channel_pool_size: int = 1,
```

Comment:
Can you plase add these options with `Annotated[..., Doc()]` declaration to `RabbitBroker.__init__` and FastAPI integration router init method too?

Reply:
Sorry for the delay, it's done!